File: Feitos em Java/2685.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        while (leitor.hasNext()) {
            int x = leitor.nextInt();
            if((x>=0&&x<90)||x==360){
                System.out.println("Bom Dia!!");
            }
            else if(x>=90&&x<180){
                System.out.println("Boa Tarde!!");
            }
            else if(x>=180&&x<270){
                System.out.println("Boa Noite!!");
            }
            else{
                System.out.println("De Madrugada!!");
            }
        }
    }
}

File: Feitos em Python 3/2685.py
while True:
    try:
        x=int(input())
        if (x>=0 and x<90) or x==360:
            print("Bom Dia!!")
        elif x>=90 and x<180:
            print("Boa Tarde!!")
        elif x>=180 and x<270:
            print("Boa Noite!!")
        else:
            print("De Madrugada!!")
    except EOFError:
        break